Examples/Docs/Sources/AsyncExamples/main.swift

```diff
@@ -86,7 +86,7 @@ private func changeStreams() throws {
         let inventory = db.collection("inventory")
 
         // Option 1: use next() to iterate
-        let next = inventory.watch(options: ChangeStreamOptions(fullDocument: .updateLookup))
+        let next = inventory.watch(options: ChangeStreamOptions(fullDocument: .other("blah")))
```

Comment:
this is an example for the docs so I think we should leave it with the original type

Reply:
Oops yea I just used this to make sure it would compile and forgot to revert